package.json

```diff
@@ -23,7 +23,7 @@
     "broccoli-merge-trees": "^1.0.0",
     "ember-cli-babel": "^6.6.0",
     "es5-shim": "~4.5.2",
-    "pixi.js": "~3.0.9"
+    "pixi.js": "~4.5.2"
```

Comment:
I'm not sure this makes sense really. I guess users of pixi might need to use specific versions of pixi and we shouldn't really dictate the version at all. Instead I guess it would make more sense to do something like

```json
"pixi.js": ">=3.0.9"
```

and let the user choose which particular version they want to use.